cms/search/tests.py

```diff
@@ -66,3 +66,27 @@ def test_promoted_search(self):
         self.assertContains(response, "Suggested result")
         self.assertContains(response, "Test Bananas Page")
         self.assertContains(response, "Plantains are similar to bananas.")
+
+
+class TestPagination(TestCase):
+    fixtures = ["fixtures/menu-fixtures.json"]
+
+    def test_no_pagination(self):
+        response = self.client.get("/search/?query=no-results")
+        self.assertNotContains(response, "Previous")
+        self.assertNotContains(response, "Next")
+
+    def test_no_prev(self):
```

Comment:
This doesn't agree with the text -- should this be `test_no_next`, and the function after `test_no_prev` ?